vignettes/analysing-n-grams.Rmd

```diff
@@ -153,11 +154,17 @@ without_wall <- research_articles %>%
 Since the amount of articles is still rather large for this demonstration, we
 could select only a few journals. Here, we will look at articles from two
 leading journals within the discipline, "Journal
-of Sociology" and "American Sociological Review". To identify articles from
+of Sociology" and "American Sociological Review". 
+
+# Filtering articles
+E.g.: It can happen that for one specific filter, one has to search in more than one column (captain obvious?)
+
+To identify articles from
 those journals, we need to take a look at the columns "journal_doi",
 "journal_jcode", and "journal_pub_id". For sociological journals in general, 
 the most common identifier is "journal_jcode". To demonstrate, we look at the
 missing proportion for each of the three variables:
+(#4)
```

Comment:
EXPLAIN what and especially why you are doing something specific before doing it! My proposed change is not really good, but discriminate between general things and specific things: "one characteristic of an article can be distributed over more than one column" vs "to filter articles by journal we need these 3 columns"